content/about.md

```diff
@@ -3,58 +3,57 @@ id: about
 title: About the Galaxy Mentorshing Network Program
 ---
 
-Galaxy Mentoring Network (GMN) program is an initiative by the Galaxy Community to build her members in open source projects.
+## About Galaxy
 
-It is run on an open enrollment basis where mentees and mentors can apply at any time of need, this is so especially because we have a lot of new members who join regularly and may immediately need someone to guide them. This is a program by Galaxy members and for Galaxy members.
+[Galaxy](https://galaxyproject.org/) is an open-source, open science platform for data integration and analysis that is deployed in hundreds of organizations around the world.
 
-## Vision
+Galaxy was originally developed to support genomic data integration and analysis, and that is still the most common domain it is used in.  However, the platform itself is domain agnostic and can be used in any research domain that uses shared data formats and is compute and data-intensive.  Galaxy is now used in related domains such as proteomics, computational chemistry and molecular modeling, as well as in domains such as climate modelling and natural language processing that are far from life science.
 
-We hope to increase the proficiency of members in the use of Galaxy tools and servers, contributing to open source projects as well as becoming active members of any of our channels and working groups.
+## Why we created the Galaxy Mentoring Network (GMN)
 
-### Benefits of the GMN program
+Galaxy is a global community of researchers, trainers, students, bioinformaticians, software developers, and infrastructure providers, all using the [Galaxy platform](https://galaxyproject.org/) to perform and support data-intensive research. There are hundreds (possibly thousands) of Galaxy deployments around the world, and tens of thousands of researchers and students use Galaxy every year. Hundreds of community members contribute to the project every year, through support, training, presentations, events, code, tools, testing, and documentation.
 
-- Mentees receive expert advice on their projects, goals, and the community.
-- Shorter learning curve through the mentorship relationship.
-- Members are empowered to acquire important social, organizational, and technical skills.
-- Mentors grow their expertise by guiding members of the community.
-- An opportunity to share experience and knowledge.
-- Learning and leadership opportunity.
+Galaxy is a sprawling ecosystem and a dispersed community. It is easy for newcomers to get overwhelmed by the many Galaxy resources and community communication channels. The Galaxy Mentor Network is established to connect newcomers with experienced community members to help prevent newcomers from getting lost.
 
-## Application Guidelines
+<div class="img-fluid mx-auto trim-p">
```

Comment:
I would remove that for now. We should avoid HTML in Markdown. There should be a way to change the class of the image directly with Markdown